core/src/main/scala/org/apache/spark/executor/ExecutorExitCode.scala

```diff
@@ -47,9 +47,9 @@ object ExecutorExitCode {
 
   def explainExitCode(exitCode: Int): String = {
     exitCode match {
-      case UNCAUGHT_EXCEPTION => "Uncaught exception"
-      case UNCAUGHT_EXCEPTION_TWICE => "Uncaught exception, and logging the exception failed"
-      case OOM => "OutOfMemoryError"
+      case UNCAUGHT_EXCEPTION => "Uncaught exception."
+      case UNCAUGHT_EXCEPTION_TWICE => "Uncaught exception, and logging the exception failed."
+      case OOM => "OutOfMemoryError."
```

Comment:
Total nit but do we need these changes? they're not sentences.
Otherwise fine.

Reply:
thanks for review, removed.